hosts

```diff
@@ -1,222 +1,353 @@
-lib-adc1.princeton.edu
-lib-adc2.princeton.edu
-[annotations_staging]
-annotations-staging1.princeton.edu
-[pas_staging]
-slavery-staging1.princeton.edu
-[pas_production]
-slavery-prod1.princeton.edu
-[cantaloupe_staging]
-iiif-staging1.princeton.edu
-[cantaloupe_production]
-libimages2.princeton.edu
-[approvals_staging]
-lib-approvals-staging1.princeton.edu
-[approvals_prod]
-lib-approvals-prod1.princeton.edu
-[perconaxdb_staging_cluster]
-maria-staging1.princeton.edu
-[percona_staging]
-maria-staging2.princeton.edu
-maria-staging3.princeton.edu
-[libwww_stage]
-library-staging1.princeton.edu
-library-staging2.princeton.edu
-[libwww_prod]
-library-prod1.princeton.edu
-library-prod3.princeton.edu
-library-prod4.princeton.edu
-[recap_stage]
-recap-www-staging1.princeton.edu
-[recap_prod]
-recap-www-prod1.princeton.edu
-[pulfalight_staging]
-pulfa3-staging1.princeton.edu
-[locator_staging]
-locator-staging1.princeton.edu
-[locator_production]
-locator-prod1.princeton.edu
-[lib_jobs_staging]
-lib-jobs-staging1.princeton.edu
-[lib_jobs_prod]
-lib-jobs-prod1.princeton.edu
-[lib_svn_staging]
-lib-svn-staging1.princeton.edu
-[libsvn]
-lib-svn-prod1.princeton.edu
-[postgresql_staging]
-lib-postgres-staging1.princeton.edu
-lib-postgres-staging2.princeton.edu
-[pgbouncer_staging]
-lib-pgbouncer-staging1.princeton.edu
-[libruby_dev]
-libruby-dev.princeton.edu
-[libruby_prod]
-libruby-prod.princeton.edu
-[libstatic_staging]
-lib-static-staging1.princeton.edu
-[libstatic_prod]
-lib-static-prod1.princeton.edu
-[mudd_staging]
-lib-mudd-staging1.princeton.edu
-[mudd_prod]
-lib-mudd-prod1.princeton.edu
-[confluence_staging]
-lib-confluence-staging1.princeton.edu
-[perconaxdb_prod_cluster]
-mariadb-prod2.princeton.edu
-[percona_prod]
-mariadb-prod1.princeton.edu
-mariadb-prod3.princeton.edu
-[library_solr_staging]
-library-solr-staging1.princeton.edu
-[library_solr_prod]
-library-solr-prod1.princeton.edu
-[repository_solr_staging]
-repository-solr-staging1.princeton.edu
-[elixir_test]
-lib-elixir-test1.princeton.edu
-[ruby_office]
-ruby-office1.princeton.edu
-[smb_serve]
-lib-smb-serve.princeton.edu
-[kennyloggin]
-kennyloggin1.princeton.edu
-kennyloggin2.princeton.edu
-kennyloggin3.princeton.edu
-[gcp_postgresql_staging]
-gcp_postgres_staging1 ansible_host=10.132.0.2
-gcp_postgres_staging2 ansible_host=10.132.0.3
-[gcp_postgresql_production]
-gcp_postgres_prod1 ansible_host=10.240.0.2
-gcp_postgres_prod2 ansible_host=10.240.0.3
-[researchdata_staging]
-prds-staging1
-[researchdata_prod]
-prds-prod1
-[gcp_oar_staging]
-gcp_oar_staging1 ansible_host=10.132.0.4
-[gcp_oar_production]
-gcp_oar_prod1 ansible_host=10.244.0.3
-[gcp_oar_dev]
-gcp_oar_dev1 ansible_host=10.0.20.3
-[gcp_dataspace_dev]
-gcp_dataspace_dev1 ansible_host=10.0.20.4
-[gcp_dataspace_staging]
-gcp_dataspace_staging1 ansible_host=10.132.0.5
-[gcp_dataspace_production]
-gcp_dataspace_prod1 ansible_host=10.244.0.2
+---
+all:
+  children:
+	bibdata_staging:
+	  hosts:
+		bibdata-staging1.princeton.edu:
```

Comment:
@kayiwa I just opened an issue for that. We use only bibdata-staging2. https://github.com/pulibrary/princeton_ansible/issues/1822

Reply:
I don't know if there is another reason that we should keep bibdata-staging1.